core/src/modules/plugin.ts

```diff
@@ -75,6 +76,14 @@ export namespace plugin {
     });
   }
 
+  export async function cleanSettings(staleKeys: string[]) {
```

Comment:
I think this name could be better. I think it's `inUseKeys`.

Reply:
Great catch! Fixed.